go/service/teams.go

```diff
@@ -222,8 +222,17 @@ func (h *TeamsHandler) TeamAddMember(ctx context.Context, arg keybase1.TeamAddMe
 		return result, nil
 	}
 
-	result.ChatSent = teams.SendTeamChatWelcomeMessage(ctx, h.G().ExternalG(), arg.Name,
-		result.User.Username)
+	result.ChatSending = true
+	go func() {
+		ctx := libkb.WithLogTag(context.Background(), "BG")
+		err := teams.SendTeamChatWelcomeMessage(ctx, h.G().ExternalG(), arg.Name,
+			result.User.Username)
+		if err != nil {
+			h.G().Log.CWarningf(ctx, "send team welcome message: %v", err)
```

Comment:
I would make this `Debug`. If it is in a goroutine its not like the user will see it as part of their CLI experience anyway, it is just going to show up at some point in time.